src/components/VerticalTabs/VerticalTabs.tsx

```diff
@@ -0,0 +1,209 @@
+	VerticalListMenuDumb as VerticalListMenu,
+} from '../VerticalListMenu/VerticalListMenu';
+
+const cx = lucidClassNames.bind('&-VerticalTabs');
+
+const { string, number, bool, func } = PropTypes;
+
+/** Vertical Tabs Tab Child Component */
+interface IVerticalTabsTabProps extends StandardProps {
+	/** Determines if the Tab is selected */
+	isSelected?: boolean;
+
+	/** Custom title for the tab */
+	Title?: string;
+}
+
+const Tab = (_props: IVerticalTabsTabProps): null => null;
+
+Tab.displayName = 'VerticalTabs.Tab';
+Tab.peek = {
+	description: `
+        Content that will be rendered in a tab. Be sure to nest a Title
+        inside each Tab or provide it as a prop.
+    `,
+};
+Tab.propName = 'Tab';
+Tab.propTypes = {
+	isSelected: bool`
+        Determines if the Tab is selected.
+    `,
+};
+
+/** Vertical Tabs Title Child Component */
+interface IVerticalTabsTitleProps extends StandardProps {}
+
+const Title = (_props: IVerticalTabsTitleProps): null => null;
+
+Title.displayName = 'VerticalTabs.Title';
+Title.peek = {
+	description: `
+        Titles can be provided as a child or prop to a Tab.
+    `,
+};
+Title.propName = 'Title';
+
+/** Vertical Tabs Component */
+interface IVerticalTabsPropsRaw extends StandardProps {
+	/** Custom Tab component (alias for `VerticalTabs.Tab`) */
+	Tab?: React.ReactNode;
+
+	/** Custom Title component (alias for `VerticalTabs.Title`) */
+	Title?: React.ReactNode;
+
+	/** Indicates which of the \`VerticalTabs.Tab\` children is currently
+    selected */
+	selectedIndex?: number;
+
+	/** Callback fired when the user selects a \`VerticalListMenu.Item\`.*/
+	onSelect: (
+		index: number,
+		{
+			event,
+			props,
+		}: {
+			event: React.MouseEvent;
+			props: IVerticalListMenuItemProps;
+		}
+	) => void;
+}
+
+type IVerticalTabsProps = Overwrite<
+	React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+	IVerticalTabsPropsRaw
+>;
+
+/** Default props for the VerticalTabs component */
+const defaultProps = {
+	selectedIndex: 0,
+	onSelect: _.noop,
+};
+
+export interface IVerticalTabsState {
+	selectedIndex: number;
+}
+
+class VerticalTabs extends React.Component<
+	IVerticalTabsProps,
+	IVerticalTabsState
+> {
+	static displayName = 'VerticalTabs';
+	static propTypes = {
+		className: string`
+			Class names that are appended to the defaults.
+		`,
+
+		selectedIndex: number`
+			Indicates which of the \`VerticalTabs.Tab\` children is currently
+			selected. The index of the last \`VerticalTabs.Tab\` child with
+			\`isSelected\` equal to \`true\` takes precedence over this prop.
+		`,
+
+		onSelect: func`
+			Callback for when the user clicks a tab. Called with the index of the tab
+			that was clicked.  Signature: \`(index, { event, props}) => {}\`
+		`,
+	};
+
+	static defaultProps = defaultProps;
+	static reducers = reducers;
+	static Tab = Tab;
+	static Title = Title;
+
+	// For backward compatibility with buildHybridComponent
+	static definition = {
```

Comment:
You can delete this `definition` static and simply move the `peek` property to a new, top-level static instead as seen here: https://github.com/appnexus/lucid/blob/de3aa74d866a04c4d6a09f591f5c753f855dae76/src/components/ToolTip/ToolTip.tsx#L145

Reply:
done!